inference-engine/cmake/developer_package_config.cmake.in

```diff
@@ -20,10 +20,14 @@ endforeach()
 message(STATUS "The following CMake options are exported from Inference Engine Developer package")
 message("")
 foreach(option IN LISTS ie_options)
-	message("    ${option}: ${${option}}")
+    message("    ${option}: ${${option}}")
 endforeach()
 message("")
 
+if(EXISTS "${CMAKE_TOOLCHAIN_FILE}")
+    include("${CMAKE_TOOLCHAIN_FILE}")
+endif()
```

Comment:
Do we need to explicitly include `CMAKE_TOOLCHAIN_FILE`? As far as I know, CMake processes `CMAKE_TOOLCHAIN_FILE` option internally.

Reply:
it processes internally once it's passed as CMD option. Just defining a variable has no effect (tested manually)

Reply:
But, in general case the toolchain file sets such variables, like `CMAKE_CXX_COMPILER`. This variable can be set only once. In this case the CMake for child project will first set the variable to some default value (system compiler) and then will try to override it when the child script will call `find_package(InferenceEgnineDeveloper)`. It will fail, won't it?

Reply:
> This variable can be set only once

Why? I can override it for any particular target / directory

> In this case the CMake for child project will first set the variable to some default value

What do you mean by child projects? Can you provide a more detailed flow when something can be broken?